Externals/wxWidgets3/include/wx/bitmap.h

```diff
@@ -305,13 +302,7 @@ wxBitmap::
 #endif
 ConvertToDisabled(unsigned char brightness) const
 {
-    // XXX comex: scale support
-    wxImage disabledImage = ConvertToImage().ConvertToDisabled(brightness);
-    #ifdef __APPLE__
-    return wxBitmap(disabledImage, -1, GetScaleFactor());
-    #else
-    return disabledImage;
-    #endif
+    return ConvertToImage().ConvertToDisabled(brightness);
```

Comment:
Is this change intentional? comex has done some wx hacks in commits like 693cee36 and 18abc333